backend/test/Test/Setup.hs

```diff
@@ -44,19 +50,36 @@ postgresTestServerConnString = lookupEnv postgresTestServerEnvName >>= \case
 -- | Provide 'EdnaContext' to a spec. It's based on the default config,
 -- but uses a custom connection string specifically for tests.
 -- It initializes DB and resets it in the end.
-withContext :: SpecWith EdnaContext -> Spec
-withContext = around withContext'
-  where
-    withContext' :: (EdnaContext -> IO a) -> IO a
-    withContext' callback = do
-      connString <- postgresTestServerConnString
-      let testConfig = defaultEdnaConfig &
-            ecDb . dbInitialisation ?~ DbInit EnableWithDrop "./sql/init.sql" &
-            ecDb . dbConnString .~ connString &
-            ecLogging .~ LogNothing
-      runEdna testConfig $ do
-        ctx <- ask
-        liftIO $ callback ctx
+specWithContext :: SpecWith EdnaContext -> Spec
+specWithContext = around withContext
```

Comment:
Btw, here we can probably use `aroundAll`, too. We don't need to create `EdnaContext` for each individual test, we can create it once for all tests. But I didn't test it, maybe something will go wrong if we do it.

Reply:
I'll check it